docs/0001-digest-algorithms.md

```diff
@@ -0,0 +1,25 @@
+# OCFL Community Extension 0001: Digest Algorithms
+
+  * Authors: OCFL Editors
+  * Minimum OCFL Version: 1.0
+  * Obsoletes: n/a
+  * Obsoleted by: n/a
+
+## Overview
+
+This extension is an index of additional digest algorithms that are defined in community extensions. It provides a controlled vocabulary of digest algorithm names that may be used to indicate the given algorithm in `fixity` blocks of OCFL Objects, and links their defining extensions.
```

Comment:
Is this phrase redundant?
``` 
...that are defined in community extensions.
```

Reply:
Hangover from earlier iteration, removed